_assets/js/main.js

```diff
@@ -875,6 +875,10 @@ jQuery(document).ready(function($) {
             if (e.key == 'Escape' && $panel.is(':visible')) {
               closePanel($link, $panel);
             }
+          })
+          .on('scroll', function() {
+            closePanel($link, $panel);
+            $(document).off('scroll');
```

Comment:
This should probably be called in `closePanel` instead (don't forget to use the namespace).